pkg/api/validation/validation.go

```diff
@@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	return allErrs
```

Comment:
Would it be possible to e.g. use Cinder-compliant volume plugins? This would permit more flexible implementation, and volume providers don't have to rewrite their plugins. It is a bit out of the scope of this PR.